students/ryba_paula/lesson_10_organizing_files/selective_copy.py

```diff
@@ -0,0 +1,28 @@
+import os
+import shutil
+
+
+file_type = ".jpg"
+source = "/home/rybaapau/test"
+destination = "/home/rybaapau/test_copy/"
+
+
+def new_folder():
+    new = destination
+    if not os.path.exists(new):
+        os.makedirs(new)
+
+
+def selective_copy(folder):
+    for foldername, subfolders, filenames in os.walk(folder):
+        for filename in filenames:
+            if filename.lower().endswith(file_type):
+                filename_path = os.path.join(foldername, filename)
+                shutil.copy(filename_path, destination)
```

Comment:
`shutil` zawiera również funkcje `copy`, od zwykłego `copy2` różni się tym, że dodatkowo kopiuje metadane pliku (o ile jest to możliwe)